### DISCUSSION TITLE
Manual multi material filament change option.

### DISCUSSION THREAD

**Discussion by Depep1** [OrcaSlicer/OrcaSlicer#4518] [Ideas]
I recently wanted to print a tpu model that would need support on only 1 layer. Could this become an option for people who want to print multi-material models with no ams or materials incompatible with an ams? 

How I imagine it would work is by running a filament change command mid-layer.

**Comment by Henelik** [comment]
+++
Theoretically, just allowing users to change the command that runs at multimaterial filament changes would solve this.
